roles/openshift_logging/tasks/generate_clusterroles.yaml

```diff
@@ -14,3 +14,14 @@
           - delete
           - update
   check_mode: no
+
+- name: Generate ClusterRole for cluster-reader
+  template: src=clusterrole.j2 dest={{mktemp.stdout}}/templates/logging-{{obj_name}}-clusterrole.yaml
+  vars:
+    obj_name: rolebinding-reader
+    rules:
+      - resources: [clusterrolebindings]
+        verbs:
+          - get
+  when: es_ops_allow_cluster_reader 
```

Comment:
we actually need to run this all the time, this is a byproduct of us checking for cluster-reader... the ES plugin now needs to do a rolebinding lookup to get users in the 'cluster-admins' and (if necessary) 'cluster-readers' roles